### PR TITLE
Icon fixes

### PR DESCRIPTION
**Pull request by mehalshah**

### CODE REVIEW

File: apps/src/code-studio/components/progress/progress_dot.jsx [hunk 1]
```diff
@@ -186,7 +188,8 @@ export const ProgressDot = React.createClass({
     const smallDot = !this.props.courseOverviewPage && !onCurrent;
     const showLevelName = /(named_level|peer_review)/.test(level.kind) && this.props.courseOverviewPage;
     const isPeerReview = level.kind === 'peer_review';
-    const iconForLevelStatus = !isUnplugged && this.props.courseOverviewPage && this.getIconForLevelStatus(level);
+    const iconForLevelStatus = !isUnplugged && showLevelName && this.props.courseOverviewPage && this.getIconForLevelStatus(level);
+    console.log('icon is ' + iconForLevelStatus);
```
Comment (aoby) [inline apps/src/code-studio/components/progress/progress_dot.jsx]: leftover trace log?

File: dashboard/test/controllers/plc/user_course_enrollments_controller_test.rb [hunk 2]
```diff
@@ -97,6 +97,29 @@ class Plc::UserCourseEnrollmentsControllerTest < ActionController::TestCase
...
+    sign_out @user
+    sign_in @district_contact
+    get :index
+    assert_response :success
+    get :group_view
+    assert_response :success
+    assert_select 'table tbody tr', 2
+    get :manager_view, id: teacher1.plc_enrollments.first
+    assert_response :success
+  end
```
Comment (aoby) [inline dashboard/test/controllers/plc/user_course_enrollments_controller_test.rb]: Is this test related to the icon changes?

Comment (mehalshah) [reply dashboard/test/controllers/plc/user_course_enrollments_controller_test.rb]: No - reverted by mistake, will remove

**Comment by aoby** [comment]
LGTM
